suite-native/module-dev-utils/src/fixtures.ts

```diff
@@ -0,0 +1,12 @@
+export const transactionNotifications = [
```

Comment:
nit: if you're using it only in demo screen, you can move it there without this new file to have the least amount of imports there so it's easier to delete once we switch to some better solution for component demo, but it doesn't really matter that much

Reply:
I did it because I think that the demo screen file is already too bloated.

Reply:
Yeah, it's not too pretty 😀